x-pack/filebeat/module/suricata/eve/test/eve-alerts.log-expected.json

```diff
@@ -17,6 +17,7 @@
         "http.request.method": "GET", 
         "http.response.status_code": "200", 
         "input.type": "log", 
+        "log.file.path": "/Users/ruflin/Dev/gopath/src/github.com/elastic/beats/x-pack/filebeat/module/suricata/eve/test/eve-alerts.log", 
```

Comment:
These paths are still present in x-pack

Reply:
will update these files too and push again.